src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class LottoTicket {
+
+    public static final int LOTTO_SIZE = 6;
+
+    private final List<Integer> lottoNumbers;
```

Comment:
하나의 로또 번호인 `Integer`를 `LottoNumber`와 같은 클래스로 포장해보는 건 어떨까요?
```suggestion
    private final List<LottoNumber> lottoNumbers;
```